packages/account/src/Components/trading-assessment/trading-assessment-dropdown.jsx

```diff
@@ -32,26 +33,24 @@ const TradingAssessmentDropdown = ({
             {item_list.map(question => (
                 <Field name={question.form_control} key={question.form_control}>
                     {() => {
+                        const has_input_error = !values[question.form_control];
                         return (
                             <React.Fragment>
                                 <DesktopWrapper>
                                     <Dropdown
-                                        classNameDisplay={classNames(
-                                            'trading-assessment__wrapper__dropdown--mobile--display',
-                                            {
-                                                'trading-frequency--field':
-                                                    question.form_control ===
-                                                        'trading_frequency_financial_instruments' &&
-                                                    ['ID', 'FR'].includes(getLanguage()),
-                                            }
-                                        )}
+                                        classNameDisplay={classNames({
+                                            'trading-frequency--field':
+                                                question.form_control === 'trading_frequency_financial_instruments' &&
+                                                ['ID', 'FR'].includes(getLanguage()),
+                                        })}
                                         is_align_text_left
-                                        name={question?.question_text}
+                                        name={question?.form_control}
                                         placeholder={question?.question_text}
                                         list={question?.answer_options}
                                         onChange={e => onChange(e, question.form_control, setFieldValue)}
                                         value={values[question.form_control]}
                                         disabled={disabled_items.includes(question.form_control)}
+                                        error={has_error && has_input_error && localize('Please select an option')}
```

Comment:
`error={has_error && has_input_error && localize('Please select an option')}` This is being used in 2 different places, you can define and pass it around.

Also this is a Component and no need to have localize inside it, I know it might has other translations but we need to stop adding to it but rather pass them as props!